app/build.gradle

```diff
@@ -277,14 +286,20 @@ gradle.taskGraph.whenReady {
             if (!android.signingConfigs.release.storeFile.exists()) {
                 throw new IllegalStateException("To perform a release build, you must have the release repo cloned at ${releaseRepoRoot}.")
             }
-            if (System.console() == null) {
+
+            if (System.console() == null && !System.getenv("CI")) {
```

Comment:
This check is here just to ensure that we can read the password from the console, so it should get moved into the `if` block below (and the message could be massaged a bit).

Reply:
Fixed